src/main/java/ma/hibernate/model/Phone.java

```diff
@@ -72,4 +72,14 @@ public Object clone() {
             throw new RuntimeException("Can't make clone of " + this);
         }
     }
+
+    @Override
+    public String toString() {
+        return "Phone{ id=" + id
+                + ", model='" + model + '\''
+                + ", maker='" + maker + '\''
+                + ", color='" + color + '\''
+                + ", os='" + os + '\''
+                + ", countryManufactured='" + countryManufactured + " }";
+    }
```

Comment:
I guess it was used while debugging
If so, it's not unnecessary  in here
I'd, personally, remove

Reply:
You are right but it also is a right thing to do - to redefine toString, isn't it? I would rather keep it in case I would need in for the next HW...